source/AI.cpp

```diff
@@ -3439,7 +3432,7 @@ void AI::MovePlayer(Ship &ship, const PlayerInfo &player)
 	if(autoPilot.Has(Command::LAND) || (autoPilot.Has(Command::JUMP) && isWormhole))
 	{
 		if(ship.GetPlanet())
-			autoPilot.Clear();
+			activeCommands.Clear(Command::LAND | Command::JUMP);
```

Comment:
Is this right?

Reply:
No, should have been autoPilot. Changed to autoPilot.